### DISCUSSION TITLE
CVE-2023-6927  Keycloak open redirect workaround

### DISCUSSION THREAD

**Discussion by bonzo71** [keycloak/keycloak#26225] [Q&A] [answered]
Hello,

I have saw this vulnerability which impacts our Keycloak 18.0.0: https://securityblog.omegapoint.se/en/writeup-keycloak-cve-2023-6927/

Except for patching, which is still very fresh, does anyone have any workaround? I was thinking about removing a * from the security-admin-console redirect URL, but I can't test all the setups that we have in the realms.

**Comment by abstractj** [comment, marked as answer]
@bonzo71 The CVE you referred to has been addressed and resolved through: https://github.com/keycloak/keycloak/security/advisories/GHSA-9vm7-v8wj-3fqw. We highly recommend upgrading to ensure that you can take advantage of the provided patch.
